website_event_attendee_fields_custom/static/description/index.html

```diff
@@ -34,59 +34,66 @@ <h3 class="oe_slogan">Ask information on registration and stores at Partner reco
-						<div class="oe_span8">
-							<h2>Need our service?</h2>
-							<p class="oe_mt32">Contact us by <a href="mailto:it@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus" target="_blank">request form</a></p>
-							<ul>
-								<li><a href="mailto:it@it-projects.info">it@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
-								<li><a href="https://www.it-projects.info/page/website.contactus" target="_blank">https://www.it-projects.info/page/website.contactus <i class="fa fa-list-alt"></i></a></li>
-							</ul>
-						</div>
-						<div class="oe_span4">
-							<div class="stamp" style="width:200px;">
-								<div style="margin-top: 15px;
-											position: relative;
-											font-family:'Vollkorn', serif;
-											font-size: 16px;
-											line-height: 25px;
-											text-transform: uppercase;
-											font-weight: bold;
-											color: #75526b;
-											border: 3px dashed #75526b;
-											float: left;
-											padding: 4px 12px;
-											-webkit-transform: rotate(-6deg);
-											-o-transform: rotate(-6deg);
-											-moz-transform: rotate(-6deg);
-											-ms-transform: rotate(-6deg);">
-									Tested on Odoo<br>10.0
-								</div>
-							</div>
-						</div>
-					</div>
-				</section>
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span8">
+            <h2>Need our service?</h2>
+            <p class="oe_mt32">Contact us by <a href="mailto:apps@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus " target="_blank">request form</a></p>
+            <ul>
+                <li><a href="mailto:apps@it-projects.info">apps@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
+                <li><a href="https://www.it-projects.info/page/website.contactus " target="_blank">https://www.it-projects.info/page/website.contactus <i class="fa fa-list-alt"></i></a></li>
+            </ul>
+        </div>
+        <div class="oe_span4">
+            <div class="stamp" style="width:200px;">
+                <div style="margin-top: 15px;
+                            position: relative;
+                            font-family:'Vollkorn', serif;
+                            font-size: 16px;
+                            line-height: 25px;
+                            text-transform: uppercase;
+                            font-weight: bold;
+                            color: #75526b;
+                            border: 3px dashed #75526b;
+                            float: left;
+                            padding: 4px 12px;
+                            -webkit-transform: rotate(STAMP1_ROTATIONdeg);
+                            -o-transform: rotate(STAMP1_ROTATIONdeg);
+                            -moz-transform: rotate(STAMP1_ROTATIONdeg);
+                            -ms-transform: rotate(STAMP1_ROTATIONdeg);">
```

Comment:
There shall not be STAMP1_ROTATION variables. Something wrong with script execution